app/helpers/i18n_helpers.rb

```diff
@@ -48,7 +65,33 @@ def t_model(model = nil)
     else
       model_name = model.class.name.underscore
     end
-    I18n::translate(model_name, :scope => [:activerecord, :models])
+    I18n::translate(model_name, :scope => [:activerecord, :models], count: count)
+  end
+
+  # Returns translated name for the given +model+ pluralized.
+  #
+  # This is the pluralization version of t_model.
+  #
+  # There needs to be one and other keys in:
+  # de.activerecord.models.account.one = Konto
+  # de.activerecord.models.account.other = Konten
+  #
+  # Example:
+  #   t_models(Account)     => 'Konten'
+  #   t_models              => 'Konten' # when called in patients_controller views
+  #
+  def t_models(model = nil)
```

Comment:
Using 'pluralize' for english allows to skip filling in one: and other: in the english locale as well. If you pass count: 2 or count: 100 and there is `en.activerecord.models.example: Example` or nothing english wont pluralize.

But using named parameters makes sense anyway. But pardon me, as a greenhorn: If I knew that ruby had named parameters, I'm more than sure that I would have used it. ;)